### PR TITLE
fix: add cover image to og/twitter meta on customer case-study pages

### PR DESCRIPTION
**Pull request by manika-signoz**
## Pull Request

---

Case study pages miss cover images due to missing mapping, PR adds cover image to og/twitter meta on customer case-study pages

preview
Screenshot 2026-09-09 at 15 55 06

NA, observed the issue so fixing

---

### ✅ Change Type
_Select all that apply_

- [ ] ✨ Feature
- [x] 🐛 Bug fix
- [ ] ♻️ Refactor
- [ ] 🛠️ Infra / Tooling
- [ ] 🧪 Test-only

---

Missed mapping for images on case study pages

Add the field in metadata

---

- Tests added/updated: NA
- Manual verification: Done on preview
- Edge cases covered: NA

---

- Blast radius: Additive only change for image
- Potential regressions: Cover images still not showing up
- Rollback plan: Fix the issue which is present, revert is the last resort.

---

### 📋 Checklist
- [x] Tests added or explicitly not required
- [x] Manually tested
- [x] Breaking changes documented
- [x] Backward compatibility considered

---

## 👀 Notes for Reviewers

---

### CODE REVIEW

File: app/(site)/customers/[...slug]/page.tsx [hunk 1]
```diff
@@ -52,6 +52,17 @@ export async function generateMetadata(props: {
...
       const seoTitle = content?.meta_title || content?.title
 
+      let imageList = [siteMetadata.socialBanner]
+      if (content.image) {
+        imageList = typeof content.image === 'string' ? [content.image] : content.image
+      }
+
+      const ogImages = imageList.map((img) => {
+        return {
+          url: img.includes('http') ? img : siteMetadata.siteUrl + img,
```
Comment (H4ad) [inline app/(site)/customers/[...slug]/page.tsx]: Change this concatenation to be more robust (detect trailiing slash and others pitfalls), you likely have a helper in the codebase for this

Comment (manika-signoz) [reply app/(site)/customers/[...slug]/page.tsx]: done, didn't have, i created one and reused across the duplicated places, the same method was copied

File: app/(site)/customers/[...slug]/page.tsx [hunk 2]
```diff
@@ -52,6 +52,17 @@ export async function generateMetadata(props: {
...
+        }
+      })
```
[the remark this answers is not in the document]
Comment (manika-signoz) [reply app/(site)/customers/[...slug]/page.tsx]: done

[the remark this answers is not in the document]
Comment (manika-signoz) [reply app/(site)/customers/[...slug]/page.tsx]: fixed

**Review by H4ad — APPROVED** [review]
